### PR TITLE
Fix runtime error selecting subcategory on services page

### PR DESCRIPTION
**Pull request by spadgett**

### CODE REVIEW

File: app/pages/services/servicesPageController.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class ServicesPageController {
         group: 'servicecatalog.k8s.io',
         resource: 'serviceclasses'
       }, {}, (resources: any) => {
-        this.ctrl.services = resources;
+        this.ctrl.services = resources.by("metadata.name");
         this.ctrl.categories = this.constants.SERVICE_CATALOG_CATEGORIES;
```
Comment (dtaylor113) [inline app/pages/services/servicesPageController.ts]: This is fixed in PR #19

File: src/components/services-view/services-view.controller.ts [hunk 2]
```diff
@@ -81,7 +81,7 @@ export class ServicesViewController implements angular.IController {
       this.ctrl.currentSubFilter = null;
       this.ctrl.expandSubCatRow = ServicesViewController.NO_SUBCATEGORY_ROW;
     } else {
-      this.ctrl.filterByCategory(this.ctrl.currentFilter, subCategory, false);
+      this.filterByCategory(this.ctrl.currentFilter, subCategory, false);
```
Comment (dtaylor113) [inline src/components/services-view/services-view.controller.ts]: 'this.ctrl.filterByCategory' is in my code base, seems to work, and I don't get a runtime exception?

Comment (spadgett) [inline src/components/services-view/services-view.controller.ts]: Not the cause of the exception, just cleaning this up

**Comment by spadgett** [comment]
Rebased. @jeff-phillips-18 @dtaylor113 let me know if you want to use this PR or just wait for #19

**Comment by dtaylor113** [comment]
I'll merge this.  Not sure how long #19 is going to be pending.

**Pull request merged**